.gitignore

```diff
@@ -1 +1,4 @@
-.vscode/
\ No newline at end of file
+.vscode/
+frontend/node_modules/
```

Comment:
Good catch, I should've added `node_modules` to the gitignore, like, first thing because that folder is yuge